src/client/sandbox/cookie/index.js

```diff
@@ -28,7 +29,7 @@ export default class CookieSandbox extends SandboxBase {
         const parsedCookieCopy = {};
 
         for (const prop in parsedCookie) {
-            if (parsedCookie.hasOwnProperty(prop))
+            if (nativeMethods.objectHasOwnProperty.call(parsedCookie, prop))
```

Comment:
Specify valid context for native method call:
`nativeMethods.objectHasOwnProperty.call(window.Object, parsedCookie, prop)`
there and above.